pyspark/dl/models/textclassifier/textclassifier.py

```diff
@@ -35,10 +35,10 @@ def text_to_words(review_text):
 
 
 def analyze_texts(data_rdd):
-    return data_rdd.flatMap(lambda (text, label): text_to_words(text)) \
+    return data_rdd.flatMap(lambda text_label: text_to_words(text_label[0])) \
         .map(lambda word: (word, 1)).reduceByKey(lambda a, b: a + b) \
-        .sortBy(lambda (w, c): - c).zipWithIndex() \
-        .map(lambda ((w, c), i): (w, (i + 1, c))).collect()
+        .sortBy(lambda w_c: - w_c[1]).zipWithIndex() \
+        .map(lambda w_c__i: (w_c__i[0][0], (w_c__i[1] + 1, w_c__i[0][1]))).collect()
```

Comment:
it's not easy to understand the intention via lambda. how about add a function like this:

```

def index(w_c_i):
    ((w, c), i) = w_c_i
    return (w, (i + 1, c))
```